developers/weaviate/configuration/pq-compression.md

```diff
@@ -0,0 +1,255 @@
+  
+  <TabItem value="py3" label="Python (v3)">
+     <FilteredTextBlock
+       text={PyCodeV3}
+       startMarker="# START DownloadData"
+       endMarker="# END DownloadData"
+       language="py"
+     />
+  </TabItem>
+</Tabs>  
+
+## Enable PQ compression
+
+To enable PQ compression, complete the following steps. 
+
+1. [Connect to a Weaviate instance](#step-1-connect-to-a-weaviate-instance)
+1. [Configure an initial schema without PQ](#step-2-configure-an-initial-schema-without-pq)
+1. [Load some training data](#step-3-load-some-training-data)
+1. [Enable and train PQ](#step-4-enable-and-train-pq)
+1. [Load the rest of your data](#step-5-load-the-rest-of-your-data)
+
+The next few sections work through these steps.
+
+### Step 1. Connect to a Weaviate instance
+
+Use one of the Weaviate [client libraries](/developers/weaviate/client-libraries) to connect to your instance. 
+
+After you install the client, connect to your instance.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python (v4)">
+     <FilteredTextBlock
+       text={PyCode}
+       startMarker="# START ConnectCode"
+       endMarker="# END ConnectCode"
+       language="py"
+     />
+  </TabItem>
+  
+  <TabItem value="py3" label="Python (v3)">
+     <FilteredTextBlock
+       text={PyCodeV3}
+       startMarker="# START ConnectCode"
+       endMarker="# END ConnectCode"
+       language="py"
+     />
+  </TabItem>
+</Tabs>  
+
+Weaviate returns `True` if the connection is successful. 
+
+### Step 2. Configure an initial schema without PQ
+
+Every collection in your Weaviate instance is defined by a [schema](/developers/weaviate/tutorials/schema). This example defines a collection called `Questions`. Weaviate uses this schema during your initial data load. After the initial data set is loaded, you modify this schema to enable PQ. 
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python (v4)">
+     <FilteredTextBlock
+       text={PyCode}
+       startMarker="# START InitialSchema"
+       endMarker="# END InitialSchema"
+       language="py"
+     />
+  </TabItem>
+  
+  <TabItem value="py3" label="Python (v3)">
+     <FilteredTextBlock
+       text={PyCodeV3}
+       startMarker="# START InitialSchema"
+       endMarker="# END InitialSchema"
+       language="py"
+     />
+  </TabItem>
+</Tabs> 
+
+### Step 3. Load some training data
+
+This example uses a relatively small data set to demonstrate loading data.
+
+If you are starting with a new Weaviate instance, you should load between 10,000 and 100,000 objects from your data set. You can use any of the objects in your data set. If possible, chose the objects at random so that they are [independent and identically distributed](https://en.wikipedia.org/wiki/Independent_and_identically_distributed_random_variables).
+
+If you already have data in your Weaviate instance, you can move ahead to the next step. By default Weaviate uses the first 100,000 objects in your database for the training step.  
```

Comment:
I *think* (maybe check with the experts) it might be worth adding a note here to ensure PQ isn't turned on with too many objects. I think I read that it was memory intensive.

Reply:
Updated